common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/OAuth2Strategy.java

```diff
@@ -105,59 +115,93 @@ protected HttpResponse performTokenRequest(final GenericTokenRequest request) th
 
 
     /**
-     * Construct the authorization endpoint URI based on issuer and path to the authorization endpoint
+     * Construct the authorization endpoint URI based on issuer and path to the authorization endpoint.
      * NOTE: We could look at basing this on the contents returned from the OpenID Configuration document
      *
-     * @return
+     * @return URI
      */
     protected Uri createAuthorizationUri() {
         //final Uri.Builder builder = new Uri.Builder().scheme(originalAuthority.getProtocol()).authority(host).appendPath(path);
         Uri authorizationUri = Uri.withAppendedPath(mIssuer, mAuthorizationEndpoint);
         return authorizationUri;
     }
 
+    protected String getAuthorizationEndpoint() {
```

Comment:
I'm not sure about the access modifiers of these methods -- I suspect one of a few things might happen in the future:

1. We'll make these `public`
2. We'll declare these fields `final`, initialize them in the constructor, and provider `public` getters _only_

No action required for this. Mostly a mental note. Tagging @shoatman as FYI